internal/pkg/util/fs/helper.go

```diff
@@ -134,34 +134,53 @@ func RootDir(path string) string {
 	return p
 }
 
-// EvalRelative evaluates symlinks in path relative to root path. This
-// function doesn't return error but always returns an evaluated path.
+// EvalRelative evaluates symlinks in path relative to root path, it returns
```

Comment:
After going thru the code several times, I think I understand what this means, and I'm having a hard time finding a suggestion as to how to change this so that other people can understand it.

First, path is _not_ a valid path in the host, and it looks like it _must_ be a valid path in the container. That's what I guess "relative to root path" means.

The bit about "it returns a path as if it was evaluated from chroot" means that the result of this function is a valid path rooted at rootfs (not relative to it). What I mean is that "relative to rootfs" would make me thing that the result of this function is a relative path (x/y/z, relative to /a/b/c) and in fact it's an absolute path rooted at rootfs (/x/y/z, with root being /a/b/c).

The other bit of documentation that is missing here is why this is needed, or in other words, when would I need to use this? What I mean is that if path is valid in the container, why would I need to fudge with it at all? I _suspect_ that part of the confusion comes from the name of this function...